content/docs/community/contribution-guide.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Documentation contribution guide
+subtitle: Learn how to contribute to the Neon documentation
+enableTableOfContents: true
+isDraft: true
+updatedOn: '2023-10-09T08:48:08.977Z'
+---
+
+This page provides guidelines for contributing to the Neon documentation. Our goal is to create an environment where our community has the information and knowledge required to confidently participate in improving the Neon documentation.
+
+## Why should you contribute?
+
+Open-source projects are always evolving. Contributing to documentation is a great way for beginners to get started in open source and for experienced developers to explain complex topics while sharing their knowledge with the community.
+
+By contributing to the Neon docs, you're helping us create a stronger learning resource for all developers. Whether you've found a typo, a section that's hard to understand, or you've noticed that a certain topic is missing, your contribution is always welcome and appreciated.
+
+## How to contribute
+
+The content for the documentation is located in the [neondatabase/website](https://github.com/neondatabase/website) repository, in the `/content/docs` directory. To contribute, you have two options:
+
+1. Edit files directly on GitHub.
+2. Fork the [neondatabase/website](https://github.com/neondatabase/website) repository, create a branch for your changes, and submit a pull request.
+
+If you prefer the first option, which is great for edits and small updates, there is an **Edit this page** link at the bottom of each Neon documentation page.
```

Comment:
FYI, I noted in the "Doc portal" [Notion page ](https://www.notion.so/neondatabase/Docs-Portal-Design-Phase-3-13ccf42119bb4a7c8697ecce2b44cfb6) that we might want to move this Edit link (or a simpler icon) to the top of the page at some point, if we want to encourage more contributors.